### DISCUSSION TITLE
How can I use tlsAllowInvalidHostnames option of mongoDB?

### DISCUSSION THREAD

**Discussion by chunlinwang** [prisma/prisma#14431] [Q&A]
For some reason, I must use the option tlsAllowInvalidHostnames for the connection of MongoDb.

But when I used tlsAllowInvalidHostnames=true in the connection url, I have an error. 
 
`The provided database string is invalid. An invalid argument was provided: tlsallowinvalidhostnames is an invalid option. An option with a similar name exists: tlsallowinvalidcertificates in database URL. Please refer to the documentation in https://www.prisma.io/docs/reference/database-reference/connection-urls for constructing a correct connection string. In some cases, certain characters must be escaped. Please check the string for any illegal characters.
`
Do we have a way to use the option tlsAllowInvalidHostnames?

Thx.

**Comment by janpio** [comment]
Did you try using the lowercase variant `tlsallowinvalidcertificates` as the error message suggests? It should have the same impact.

**Reply by vladi-stevanovic** [reply]
👋 Hello @chunlinwang I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
